catalog/app/containers/Bucket/PackageList.js

```diff
@@ -127,9 +122,38 @@ const usePackageStyles = M.makeStyles((t) => ({
     paddingTop: t.spacing(2),
     textOverflow: 'ellipsis',
   },
+  handle: {
+    fontSize: t.typography.pxToRem(16),
+    fontWeight: t.typography.fontWeightMedium,
+    lineHeight: t.typography.pxToRem(20),
+  },
+  handleText: {
+    position: 'relative',
+  },
+  handleClickArea: {
+    bottom: 0,
+    left: 0,
+    position: 'absolute',
+    right: 0,
+    top: 0,
+
+    '$handle:hover &': {
+      background: t.palette.action.hover,
+    },
+  },
+  revisions: {
+    ...t.typography.subtitle2,
+    color: t.palette.text.secondary,
+    position: 'relative',
+  },
+  updated: {
+    ...t.typography.body2,
+    color: t.palette.text.secondary,
+    position: 'relative',
+  },
 }))
 
-const Package = ({ name, modified, revisions, revisionsTruncated, bucket, views }) => {
+function Package({ name, modified, revisions, revisionsTruncated, bucket, views }) {
```

Comment:
why not use arrow functions, there is no `this` reference, so just curious

Reply:
oldschool named functions show up properly in the debugger / stack traces (not as "Anonymous") -- that's the only reason